server/xpub/entities/manuscript/helpers/empty.js

```diff
@@ -9,7 +9,6 @@ const emptyManuscript = {
   },
   opposedSeniorEditorsReason: '',
   opposedReviewingEditorsReason: '',
-  opposedReviewers: [],
```

Comment:
Why is this being removed?

Reply:
Opposed reviewers are now being stored in teams along with all other suggestions (they we left out in the initial work because their shape was different and it would have needed special handling). So there's no need for an explicit property on the manuscript or column in the table. They still exist as a separate property on the manuscript input because that matches the shape of the form.